mobile/lib/features/wallet/receive_screen.dart

```diff
@@ -215,6 +222,34 @@ class _ReceiveScreenState extends State<ReceiveScreen> {
         BitcoinAddress(
           address: _paymentRequest == null ? "" : _paymentRequest!.address,
         ),
+        const SizedBox(height: 20),
+        Visibility(
+          visible: !hasDlcChannel && balance.sats < minBalance,
+          child: Container(
+              padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 20),
+              decoration: BoxDecoration(
+                  border: Border.all(color: tenTenOnePurple),
+                  color: Colors.white,
+                  borderRadius: BorderRadius.circular(10)),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  const Icon(
+                    Icons.info,
+                    color: tenTenOnePurple,
+                    size: 22,
+                  ),
+                  const SizedBox(width: 10),
+                  Expanded(
+                    child: Text(
+                      "Receive around ${formatSats(Amount(minBalance))} to be able to trade.",
```

Comment:
🔧 This doesn't take into account how much they might already have in the on-chain wallet. I think that's okay, but I would reword the message to:

```
"You will need around $minBalance in your on-chain wallet to start trading."
```